### PR TITLE
Added Youtube tutorial on Docker

### PR DESCRIPTION
**Pull request by tarunclub**

### CODE REVIEW

**Review by TRohit20 — APPROVED** [review]
I observed that you closed a PR where I asked you to make this change and opened a new PR with the suggested change. Next time, don't open a new PR when asked to make changes, you always have the option to make changes in the branch you're working in and directly push them upstream to the PR you opened. :)

**Comment by tarunclub** [comment]
@TRohit20 sure I will keep that in mind : )

File: DevOps/Readme.md [hunk 1]
```diff
@@ -54,6 +54,7 @@
   - [Docker Tutorial - TechWorld with Nana(Youtube)](https://www.youtube.com/watch?v=3c-iBn73dDE)
   - [Learn Docker - freecodecamp(Youtube)](https://www.youtube.com/watch?v=9zUHg7xjIqQ)
   - [Docs](https://docs.docker.com/)
+  - [A practical guide on Docker - Hitesh Choudhary(Youtube)](https://www.youtube.com/watch?v=rr9cI4u1_88&t=4594s)
```
Comment (siddhant-khisty) [inline DevOps/Readme.md:57]:
```suggestion
  - [A practical guide on Docker - Hitesh Choudhary(Youtube)](https://www.youtube.com/watch?v=rr9cI4u1_88)
```
Remove the timestamp

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

**Review by Sandeep-RN — CHANGES_REQUESTED** [review]
And I think we can also use the full name of the video "A practical guide on Docker with projects" it will be helpful for those people who are searching for Docker projects as well.

**Pull request closed without merging**